README.md

```diff
@@ -1,154 +1,187 @@
 Table of Contents
 =================
 
-  * [How to use](#how-to-use)
-    * [Monolithic Install](#monolithic-install)
-    * [Split Install ( Running on the Master )](#split-install--running-on-the-master-)
-    * [Monolithic With Compile Masters ( Running on the MoM )](#monolithic-with-compile-masters--running-on-the-mom-)
-    * [Split With Compile Masters ( Running on the MoM )](#split-with-compile-masters--running-on-the-mom-)
-    * [Running on PE 3\.8](#running-on-pe-38)
-    * [Other Option](#other-option)
-  * [What do you get](#what-do-you-get)
-    * [Grepping for Metrics](#grepping-for-metrics)
-      * [Puppetserver](#puppetserver)
-      * [PuppetDB](#puppetdb)
+- [How to use](#how-to-use)
```

Comment:
not obvious but I'm generating this from gh-md-toc.  https://github.com/ekalinin/github-markdown-toc